tests/aws/services/cloudformation/resources/test_dynamodb.py

```diff
@@ -137,3 +140,26 @@ def test_global_table(deploy_cfn_template, snapshot, aws_client):
 
     error_code = ex.value.response["Error"]["Code"]
     assert "ResourceNotFoundException" == error_code
+
+
+@markers.aws.validated
+def test_ttl_cdk(aws_client, snapshot, infrastructure_setup):
+    infra = infrastructure_setup(namespace="DDBTableTTL")
+    stack = cdk.Stack(infra.cdk_app, "DDBStackTTL")
+
+    table = dynamodb.Table(
+        stack,
+        id="Table",
+        billing_mode=BillingMode.PAY_PER_REQUEST,
+        partition_key=dynamodb.Attribute(name="id", type=dynamodb.AttributeType.STRING),
+        removal_policy=cdk.RemovalPolicy.RETAIN,
+        time_to_live_attribute="expire_at",
+    )
+
+    cdk.CfnOutput(stack, "TableName", value=table.table_name)
+
+    with infra.provisioner() as prov:
+        outputs = prov.get_stack_outputs(stack_name="DDBStackTTL")
+        table_name = outputs["TableName"]
+        table = aws_client.dynamodb.describe_time_to_live(TableName=table_name)
+        snapshot.match("table", table)
```

Comment:
question: do we have coverage of the TTL feature itself? I realise
* this test covers the CFn part, and the TTL behaviour should be tested with the service tests, and
* we rely on a third party package for the implementation so we should defer to the tests of that implementation,
but I still think it would be good to. have.

Reply:
Yup, we do have tests that cover TTL itself in the DDB suite.